### PR TITLE
5 packages from owlbarn/owl at 0.6.0

### PR DESCRIPTION
**Pull request by mseri**
This pull-request concerns:
-`owl.0.6.0`: OCaml Scientific and Engineering Computing
-`owl-base.0.6.0`: OCaml Scientific and Engineering Computing - Base
-`owl-plplot.0.6.0`: OCaml Scientific and Engineering Computing
-`owl-top.0.6.0`: OCaml Scientific and Engineering Computing - Top
-`owl-zoo.0.6.0`: OCaml Scientific and Engineering Computing - Zoo

---
* Homepage: https://github.com/owlbarn/owl
* Source repo: git+https://github.com/owlbarn/owl.git
* Bug tracker: https://github.com/owlbarn/owl/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/owl-base/owl-base.0.6.0/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+description: "Owl is an OCaml scientific library."
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "base-bigarray"
+  "dune" {build}
```
Comment (mseri) [inline packages/owl-base/owl-base.0.6.0/opam]:
```suggestion
  "dune"
```

File: packages/owl-plplot/owl-plplot.0.6.0/opam [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+"""
+
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "dune" {build}
```
Comment (mseri) [inline packages/owl-plplot/owl-plplot.0.6.0/opam]:
```suggestion
  "dune"
```

File: packages/owl-top/owl-top.0.6.0/opam [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+description: "Owl is an OCaml scientific library."
+
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "dune" {build}
```
Comment (mseri) [inline packages/owl-top/owl-top.0.6.0/opam]:
```suggestion
  "dune"
```

File: packages/owl-zoo/owl-zoo.0.6.0/opam [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+"""
+
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "dune" {build}
```
Comment (mseri) [inline packages/owl-zoo/owl-zoo.0.6.0/opam]:
```suggestion
  "dune"
```

File: packages/owl/owl.0.6.0/opam [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "alcotest" {with-test}
+  "base" {build}
+  "base-bigarray"
+  "conf-openblas" {>= "0.2.0"}
+  "ctypes"
+  "dune" {build & >= "1.2.1"}
```
Comment (mseri) [inline packages/owl/owl.0.6.0/opam]:
```suggestion
  "dune" {>= "1.2.1"}
```

**Comment by mseri** [comment]
The CI failures are the expected ones. Travis failure for 4.08 is surprising as I thought there is a ctypes version compatible with 4.08 (and as it compiles on my machine)

**Comment by kit-ty-kate** [comment]
Looks ok. Thanks!

**Pull request merged**